kubernetes/README.md

```diff
@@ -57,11 +48,15 @@ etcd:
 
 #### Create configmap for apache apisix
 
+you will change some config by use `apisix-gw-config-cm.yaml`
```

Comment:
It seems that we can always create configmap `apisix-gw-config.yaml` via `config.yaml`.
If you use the default one, you can use `config.yaml`.
If you need to change the default, you can change the `config.yaml`, so the behavior is consistent between k8s and normal installation.
By this way, we can ensure only one place hold the customized configure (for instance, only one place to change the etcd address).

Reply:
yes, you are right